docs/sources/flow/getting-started/collect-prometheus-metrics.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Collect Prometheus metrics
+weight: 200
+---
+
+# Collect Prometheus metrics
+
+Grafana Agent Flow can be configured to collect [Prometheus][] metrics and
+forward them to any Prometheus-compatible database.
+
+This topic describes how to:
+
+* Configure metrics delivery
+
+[Prometheus]: https://prometheus.io
+
+## Components used in this topic
+
+* [prometheus.remote_write][]
+* [prometheus.scrape][]
+
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+[prometheus.scrape]: {{< relref "../reference/components/prometheus.scrape.md" >}}
+
+## Before you begin
+
+* Ensure that you basic familiarity with instrumenting applications with
+  Prometheus.
+* Have a set of Prometheus exports or applications exposing Prometheus metrics
+  that you want to collect metrics from.
+* Identify where you will write collected metrics, such as Prometheus, Grafana
+  Mimir, Grafana Cloud, or Grafana Enterprise Metrics.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+
+## Configure metrics delivery
+
+Before components can collect Prometheus metrics, you must have a component
+responsible for writing those metrics somewhere.
+
+The [prometheus.remote_write][] component is responsible for delivering
+Prometheus metrics to one or Prometheus-compatible endpoints. Once a
+`prometheus.remote_write` component is defined, other Grafana Agent Flow
+components can be used to forward metrics to it.
+
+To configure a `prometheus.remote_write` component for metrics delivery,
+complete the following steps:
+
+1. Add the following `prometheus.remote_write` component to your configuration file:
+
+   ```river
+   prometheus.remote_write "LABEL" {
+     endpoint {
+       url = "PROMETHEUS_URL"
+     }
+   }
+   ```
+
+2. Replace `LABEL` with a label to use for the component, such as `default`.
+   The label chosen must be unique across all `prometheus.remote_write`
+   components in the same configuration file.
+
+3. Replace `PROMETHEUS_URL` with the full URL of the Prometheus-compatible
+   endpoint where metrics will be sent, such as
+   `https://prometheus-us-central1.grafana.net/api/prom/push`.
+
+4. If your endpoint requires basic authentication, complete the following:
+
+    1. Paste the following inside of the `endpoint` block:
```

Comment:
Should we call this 4.1, so that we can refer to it more easily if needed? And similar for other sub-points.

Reply:
No, that won't work in our Markdown.